front-end/slothstagram/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "slothstagram",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.1.1",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
```

Comment:
Are we using all of these dependencies? For instance, Jest. Let's add only what we need